pytest.ini

```
@@ -14,3 +14,4 @@ testpaths = tests
 markers =
     needs_vtk9: skip test unless VTK is 9.0 or newer.
     needs_vtk_version(version): skip test unless VTK version is at least as specified.
+    skip_download
```

Semantically I think this should be something like 
```suggestion
    needs_download: this test downloads data during execution
```
because the "skip" part comes from the new `--test_downloads` flag. What that should do is skip downloads that are marked as "needs download" (or "has download" or "downloads" or whatever). So my point is that skip should not be in the name of the marker. (And I'd definitely recommend using a description for the marker, not necessarily the one I wrote above).